docs/management_commands.rst

```diff
@@ -21,3 +23,43 @@ To prevent the CPU and RAM high peaks during deletion process use ``CLEAR_EXPIRE
 
 Note: Refresh tokens need to expire before AccessTokens can be removed from the
 database. Using ``cleartokens`` without ``REFRESH_TOKEN_EXPIRE_SECONDS`` has limited effect.
+
```

Comment:
Ugh, I forgot this is RsT and not markdown so my suggestion using triple-backticks wasn't so helpful....

Reply:
I pushed a commit with the required indentation.